packages/javascript/bh-shared-ui/src/views/Explore/EdgeInfo/EdgeInfoHeader.tsx

```diff
@@ -42,12 +41,8 @@ const Header: React.FC<HeaderProps> = ({ name = 'None Selected', onToggleExpande
 
     return (
         <div className={styles.header}>
-            <Icon
-                className={styles.icon}
-                click={() => {
-                    onToggleExpanded(!expanded);
-                }}>
-                <FontAwesomeIcon icon={expanded ? faMinus : faPlus} />
+            <Icon className={styles.icon} click={clearSelectedItem}>
```

Comment:
IIRC every time we call `clearSelectedItem` (calling `setExploreParams` underneath) we will be adding to the url history stack. For example if I click the X twice, I would need to hit the back button twice to go back to the previous state before clearing the selected item. Im thinking clearing the `selectedItem` should only update the url history once!

There are a couple ways to counter this. For future us, we could add a check to `clearSelectedItem` so we only call setExploreParams `if (!!selectedItem)` and that would solve the extra url history! And i think a potential UI improvement would be to hide this icon/button if there is no `selectedItem` (perhaps just leave empty space where the button would be?)